src/core/ext/filters/http/server/http_server_filter.h

```diff
@@ -26,10 +26,10 @@
 /* Processes metadata on the server side for HTTP2 transports */
 extern const grpc_channel_filter grpc_http_server_filter;
 
-namespace grpc_core {
-// Temporary code that allows servers to accept PUT requests. DO NOT USE WITHOUT
-// PERMISSION.
-void InternalOnlyDoNotUseUnlessYouHavePermissionFromGrpcTeamAllowBrokenPutRequests();
-}  // namespace grpc_core
+// A Temporary channel arg that allows servers to accept PUT requests. DO NOT
+// USE WITHOUT PERMISSION.
+#define GRPC_ARG_INTERNAL_ONLY_DO_NOT_USE_UNLESS_YOU_HAVE_PERMISSION_FROM_ALLOW_GRPC_TEAM_ALLOW_BROKEN_PUT_REQUESTS \
```

Comment:
We'll probably need some visibility hook on the `grpc_http_filters` BUILD target to allow us to control visibility internally.

Reply:
@ananda1066 Can you send a PR/cl for this please?

Reply:
I think the way we do this is by adding a visibility tag like the following to the BUILD target:

https://github.com/grpc/grpc/blob/55e870dae61ead220e0c74213410d90f8d21179f/BUILD#L715

We then provide a definition for that tag here:

https://github.com/grpc/grpc/blob/55e870dae61ead220e0c74213410d90f8d21179f/bazel/grpc_build_system.bzl#L103

And we can provide a different definition for the same tag internally.

We should probably add the OSS pieces here as part of this PR.

Reply:
done!